File: cdattg_web_frontend/src/pages/carnets/carnetRegional.test.ts
/**
 * Pruebo la etiqueta Regional. Guaviare.
 *
 * @author Cristian Deysdayr Jiménez
 */
import { describe, expect, it } from 'vitest';
import { etiquetaRegionalCarnet } from './carnetRegional';

describe('etiquetaRegionalCarnet', () => {
  it('antepone Regional. si solo viene Guaviare', () => {
    expect(etiquetaRegionalCarnet('Guaviare')).toBe('Regional. Guaviare');
  });

  it('no duplica la palabra Regional', () => {
    expect(etiquetaRegionalCarnet('Regional Guaviare')).toBe('Regional. Guaviare');
  });
});

File: cdattg_web_golang/services/carnet_regional_test.go
/**
 * Pruebo Regional. Guaviare.
 *
 * @author Cristian Deysdayr Jiménez
 */
package services

import "testing"

func TestEtiquetaRegionalCarnet(t *testing.T) {
	t.Parallel()
	if got := etiquetaRegionalCarnet("Guaviare"); got != "Regional. Guaviare" {
		t.Fatalf("got %q", got)
	}
	if got := etiquetaRegionalCarnet("Regional Guaviare"); got != "Regional. Guaviare" {
		t.Fatalf("got %q", got)
	}
}
